downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,219 @@
+    HTTP_1_1,
+    HTTP_2
+}
+
+configurable configLib:HttpVersion & readonly httpVersion = ?;
+```
+
+The value for `httpVersion` can be provided via the `Config.toml` file or as a command-line argument as shown below.
+
+```toml
+[configUnionTypes]
+httpVersion = "HTTP_1_1"
+```
+
+The command-line argument will be as shown below.
+
+```bash
+-ChttpVersion=HTTP_1_1
+```
+
+#### Improved Configurable Variables to Support Map Types
+
+The `configurable` feature is improved to support variables with the `map` types through the TOML syntax. 
+
+For example, if the `map`-typed configurable variables are defined in the following way, 
+
+``` ballerina
+configurable map<string> admin = ?;
+
+type HttpResponse record {|
+    string method;
+    string httpVersion = "HTTP_1_0";
+    map<string> headers;
+|};
+
+configurable HttpResponse response = ?;
+configurable map<string>[] users = ?;
+```
+
+the values can be provided in the `Config.toml` as follows.
+
+```toml
+[admin]
+username = "John Doe"
+mail = "John@hotmail.com"
+location = "LK"
+
+[response]
+method = "POST"
+httpVersion = "HTTP_2_0"
+headers.Status = "200 OK"
+headers.Content-Type = "text/html"
+
+[[users]]
+username = "Jack"
+location = "UK"
+
+[[users]]
+username = "Jane"
+location = "US"
+```
+
+### Breaking Changes
+
+- The `io.ballerina.runtime.api.types.Type#getName` and `io.ballerina.runtime.api.types.Type#getQualifiedName` methods now return an empty string if no name was associated with the type. The `io.ballerina.runtime.api.types.Type#toString` method can be used to get the string representation of a type if required.
+- `Wait` actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error`in which `T` is the type of the original return value.
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `graphql` Package
+- Added declarative auth configurations
+- Added support for optional types as inputs
+- Added support for returning distinct service object unions
+- Added support for inline fragments
+- Added support for enums as input values
+
+### Improvements
+
+#### `cache` Package
+Marked the `cache:Cache` class as an isolated class
+
+#### `file` Package
+Marked the `file:Listener` class as an isolated class
+
+#### `graphql` Package
+- Improved introspection validation and execution
+- Added missing fields in the GraphQL types
+- Improved the compiler plugin to validate inputs and return types
+- Added support to use included record parameters instead of the `record` type in the listener initialization
+
+#### `http` Package
+- Improved the `http:Client` remote methods to support the contextually-expected type inference
```

Comment:
Is it the target type that we infer? Can we say something like

```suggestion
- Improved the `http:Client` remote methods to support inferring the target type from the contextually-expected type
```

@chamil321 

contextually-expected type inference isn't quite accurate IMO. We infer the argument from the contextually-expected type